src/workflows-app/components/LeftNavigationPanel.ts

```diff
@@ -0,0 +1,156 @@
+                style: { alignItems: 'center', flexDirection: 'column', paddingLeft: '2rem' },
+                role: 'list',
+              },
+              [
+                _.map(([subHeaderKey, subHeaderName]) => {
+                  const isActive = isSubHeaderActive(subHeaderKey);
+                  return h(
+                    Clickable,
+                    {
+                      'aria-label': `${subHeaderKey}-header-button`,
+                      style: { color: isActive ? colors.accent(1.1) : colors.accent(), fontSize: 16 },
+                      onClick: () => setSelectedSubHeader(subHeaderKey),
+                      hover: Style.navList.itemHover(isActive),
+                      'aria-current': isActive,
+                      key: subHeaderKey,
+                    },
+                    [
+                      h(ListItem, {
+                        title: subHeaderName,
+                        style: { borderBottom: `0.5px solid ${colors.dark(0.2)}` },
+                      }),
+                    ]
+                  );
+                }, Object.entries(findAndAddSubheadersMap)),
+                div(
+                  {
+                    style: {
+                      marginRight: '3em',
+                      marginTop: '1.5em',
+                      display: 'flex',
+                      alignItems: 'center',
+                      flex: 'none',
+                      backgroundColor: colors.accent(0.06),
+                      boxShadow: '0 2px 5px 0 rgba(0,0,0,0.35), 0 3px 2px 0 rgba(0,0,0,0.12)',
+                      paddingTop: '0.5em',
+                      paddingBottom: '0.5em',
+                      paddingLeft: '0.75em',
+                      paddingRight: '0.75em',
+                      borderRadius: '15px',
+                    },
+                  },
+                  [
+                    h(
+                      Clickable,
+                      {
+                        role: 'list',
+                        href: `${
+                          getConfig().dockstoreUrlRoot
+                        }/search?_type=workflow&descriptorType=WDL&searchMode=files`,
+                      },
+                      [
+                        div({ style: { fontWeight: 'bold' } }, ['Dockstore  ', icon('export')]),
+                        div([
+                          'Browse WDL workflows in Dockstore, an open platform used by the GA4GH for sharing Docker-based workflows.',
+                        ]),
+                      ]
+                    ),
+                  ]
+                ),
+              ]
+            ),
+          ]
+        ),
+      ]),
+      h(HelpfulLinksBox, {
+        method: null,
+        style: { borderRadius: '20px', margin: '1.35em', bottom: '0', position: 'fixed' },
+      }),
+    ]),
+    isSubHeaderActive('workspace-workflows') && div(['TODO']),
+    isSubHeaderActive('submission-history') && h(SubmissionHistory, { name, namespace, workspace }),
```

Comment:
Loading the Submission History page as is is loading entire Terra UI into the right panel. I would suggest putting a TODO here as well and creating a separate ticket to import this page into the new panel.
<img src="https://github.com/DataBiosphere/terra-ui/assets/16748522/67c75538-0d4b-400f-bb8a-7f67789344a3" width="1000" height="400" />